src/cli-validator/utils/printJsonResults.js

```diff
@@ -33,6 +33,10 @@ module.exports = function printJson(results, originalFile) {
     });
   });
   // render the results to json in the console with 2 char spacing
+  if (errorsOnly) {
+    delete results.warnings;
+  }
+  // render the results to json in the console with 2 char spacing
```

Comment:
Maybe I was unclear in my last comment, but did you not notice that the comment is now present twice?
>  // render the results to json in the console with 2 char spacing